components/statuslight/index.css

```diff
@@ -10,68 +10,148 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "../vars/css/components/spectrum-statuslight.css";
+.spectrum-StatusLight {
+  /* Static tokens */
+  --spectrum-statuslight-corner-radius: 50%;
+  --spectrum-statuslight-font-weight: 400;
```

Comment:
we could use here for now something like `  --spectrum-font-weight-regular: 400;` https://github.com/adobe/spectrum-css/blob/main/components/tokens/custom-spectrum/custom-vars.css#L32